src/test/regress/expected/create_role_propagation.out

```diff
@@ -240,10 +242,11 @@ SELECT roleid::regrole::text AS role, member::regrole::text, (grantor::regrole::
       role       |     member      | grantor | admin_option
 ---------------------------------------------------------------------
  dist_role_1     | dist_role_2     | t       | f
- dist_role_3     | non_dist_role_3 | t       | f
```

Comment:
Should we change the query into this so that we can verify whether we set grantor properly?

```
SELECT roleid::regrole::text AS role, member::regrole::text, grantor::regrole::text, admin_option FROM pg_auth_members WHERE roleid::regrole::text LIKE '%dist\_%' ORDER BY 1, 2;
```

Reply:
Removed conditional option and left grantor

Reply:
we can not since grantor for pg 14 and pg 15 is different from pg16. I think that's why they put such check

Reply:
What is the difference?
There is probably some valid reason for this change but it kind of makes the test useless, since the grantor will almost always be `postgres` in our tests.

Reply:
Now added test_admin_role into viable grantor list to make grantor t for the list.